h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -2083,6 +2076,8 @@ protected Submodel computeSubmodel(int i,double lambda) {
 
     private transient double _nullDevTrain = Double.NaN;
     private transient double _nullDevTest = Double.NaN;
+    private boolean _earlyStop = false;  // set by earlyStopping
```

Comment:
this confused the hell out of me.
1) _nullDevTrain do not need to be instance variables with global scope, these can be just regular vars inside computeImpl
2) either move earlyStop to the top with the rest of the instance vars or (preferably) make it a regular var inside computeImpl

Reply:
Moved nullDevTrain/nullDevTest to inside computeimpl.  Moved earlyStop to the top.